types/address.go

```diff
@@ -133,6 +134,7 @@ func AccAddressFromHex(address string) (addr AccAddress, err error) {
 	return AccAddress(bz), err
 }
 
+// TODO make an issue to get rid of global Config
```

Comment:
I add `TODO` to make an issue to get rid of `global Config` after this PR is merged. Should I make this issue now?

Reply:
I added an item as part of #9690

Reply:
Let's reference #9690 in this comment please @cyberbono3 🙏

Reply:
Sure